skyVegasTests/utilities/makeSpinHelper.ts

```diff
@@ -3,7 +3,9 @@ import { validateConsoleMessages } from './validateConsoleMessagesHelper';
 import { ExpectedMessage } from '../../types/expectedMessage';
 
 export const makeSpin = async (page: Page, consoleMessages: string[]) => {
+    
     await page.mouse.click(300, 300, { delay: 10 });
+    await new Promise((r) => setTimeout(r, 2000));
```

Comment:
Is this needed? The bug was for the search only AFAIR

Reply:
Yeah,  this ticket shouldn't fix make spin function

Reply:
I noticed the issue as I was testing and thought to try and set a timeout for the spin and it worked after that. Or should this be sorted in a different ticket?

Reply:
I have removed this though, as we have all agreed the multiple timeouts are not the best way to sort this issue out and the ticket is just for the search bar. :)